package/gargoyle/files/www/js/access.js

```diff
@@ -59,21 +56,41 @@ function saveChanges()
 			uci.set("firewall", id, "zone", "wan");
 			firewallSectionCommands.push("uci set firewall." + id + "=remote_accept");
 		};
+		var localHttpPort = document.getElementById("local_http_port").value;
+		var remoteHttpPort = document.getElementById("remote_http_port").value;
+		if(document.getElementById("remote_http_port_container").style.display != "none")
+		{
+			addAccept(localHttpPort, remoteHttpPort);
+		}
+		else
+		{
+			remoteHttpPort = "";
+		}
+		if(oldRemoteHttpPort != remoteHttpPort) restartFirewall = true;
+		var localHttpsPort = document.getElementById("local_https_port").value;
+		var remoteHttpsPort = document.getElementById("remote_https_port").value;
 		if(document.getElementById("remote_https_port_container").style.display != "none")
 		{
-			addAccept( document.getElementById("local_https_port").value, document.getElementById("remote_https_port").value);
+			addAccept(localHttpsPort, remoteHttpsPort);
 		}
-		if(document.getElementById("remote_http_port_container").style.display != "none")
+		else
 		{
-			addAccept( document.getElementById("local_http_port").value, document.getElementById("remote_http_port").value);
+			remoteHttpsPort = "";
 		}
+		if(oldRemoteHttpsPort != remoteHttpsPort) restartFirewall = true;
```

Comment:
One line IF statements still require brackets as per the coding guidelines for Gargoyle. Please fix, or consider refactoring into the accepted `x = y == z ? A : B;`

Reply:
Yes, fixed using brackets.